mkdocs.yml

```diff
@@ -0,0 +1,107 @@
+site_name: Aeternity JavaScript SDK
+extra_css: [extra.css]
+theme:
+    name: readthedocs
+    highlightjs: true
+    hljs_languages:
+        - yaml
+        - rust
+        - erlang
+        - javascript
+        - js
+        
+markdown_extensions:
+    - attr_list
+    - admonition
+    - codehilite
+    #- pymdownx.details
+    #- pymdownx.superfences
+
+nav:
+    - Overview : 'index.md'
+    - Quick Start : './guides/quick-start.md'
+    - Guides : 
+        - Overview : './guides.md'
+        - 'In Depth' : 
+            - Events : './guides/contract-events.md'
+            - Oracles : './guides/oracle-usage.md'
+            - 'AENS usage' : './guides/aens-usage.md'
+            - 'Contract ACI' : './guides/contract-aci-usage.md'
+            - 'Contract Events' : './guides/contract-events.md'
+            - 'Delegate Signature to Contract' : './guides/delegate-signature-to-contract.md'
+            - 'Wallet Api' : './guides/how-to-build-aepp-using-new-wallet-api.md'
+            - 'Building Wallets' : './guides/how-to-build-an-wallet-app-or-extension.md'
+            - 'Low vs High level API' : './guides/low-vs-high-usage.md'
+            - 'Tree Shaking' : './guides/import-tree-shaking.md'
+            - 'Migration' : './guides/migration/migration-7.0.0.md'
+        - 'API Reference' :
```

Comment:
API reference usually is not kind of guide, I'm proposing to move it to the root.